packages/react-widgets/src/DateTimePicker.js

```diff
@@ -227,7 +227,7 @@ class DateTimePicker extends React.Component {
   constructor(...args) {
     super(...args)
 
-    this.inputId = instanceId(this, '_input')
+    this.inputId = this.props.id || instanceId(this, '_input')
```

Comment:
this will lead to two elements having the same `id` since `props.id` is also (correctly) passed to the container element here. Instead you want to use `inputProps.id` if inputProps is provided. 

Also can you make this same change in the other picker widgets that suffer from the same issue?

Reply:
`const resovledInputId = inputProps && typeof inputProps.id === 'string' ? inputProps.id : this.inputId;`

Does this sound good? I guess the `id` prop is not exposed as a prop as per documentation. 
`
<DateTimePickerInput
        {...inputProps}
        id={resovledInputId}
        // rest of the code
        />
`

Reply:
I would probably add helper to the widgetHelpers.
`const resolvedInputId = hasValidCustomInputId(inputProps.id) ? inputProps.id : this.inputId;` and try to add across other widgets.

Reply:
`inputProps?.id || createId(this)` should be fine

Reply:
Changes made so that **DateTimePicker** and **Combobox** resolves the input id in this manner. Other widgets like **Multiselect**, **NumberPicker** pass the `inputProps `directly to the input without as is.

Didn't change **Select** as this would mean, multiple ids per select option and DropdownList's input won't have the selected value and rather the search input and also its not focusable.